packages/react-native-reanimated/src/logger/logger.ts

```diff
@@ -55,27 +66,59 @@ export function logToLogBoxAndConsole(data: LogData) {
 }
 
 /**
- * Replaces the default log function with a custom implementation.
+ * Updates logger configuration.
  *
- * @param logFunction - The custom log function.
+ * @param config - The new logger configuration to apply.
+ *   - level: The minimum log level to display.
+ *   - strict: Whether to log warnings and errors that are not strict.
+ *    Defaults to false.
+ *   - logFunction: The function to use for logging.
  */
-export function replaceLoggerImplementation(
-  logFunction: (data: LogData) => void
+export function updateLoggerConfig(config?: Partial<LoggerConfigInternal>) {
+  'worklet';
+  global.__loggerConfig = {
+    logFunction:
+      // Re-use previously assigned log function if it exists
+      config?.logFunction ?? global.__loggerConfig?.logFunction ?? logToConsole,
+    // Don't reuse previous level and strict values from the global config
+    level: config?.level ?? 'warn',
+    strict: config?.strict ?? false,
+  };
```

Comment:
Why global scope? 👀

Reply:
This was global for the same reason as the `ReanimatedError` in the first PR. Because it needs to be global in the Worklet runtime, I made it global in the React runtime as well.

I will adjust changes based on your recent suggestions I already applied to previous PRs.